pkg/report/linux_test.go

```diff
@@ -230,3 +235,187 @@ func TestLinuxSymbolizeLine(t *testing.T) {
 		})
 	}
 }
+
+func prepareLinuxReporter(platform string, t *testing.T) *linux {
+	config := &config{
+		target: targets.Get(targets.Linux, platform),
+	}
+	reporter, _, err := ctorLinux(config)
+	if err != nil {
+		t.Errorf("Failed to create a reporter instance %#v", platform)
+	}
+	return reporter.(*linux)
+}
+
+// nolint:dupl
+func TestParseLinuxOpcodes(t *testing.T) {
+	type validTest struct {
```

Comment:
I think it can be made more data-driven as:
```
	type validTest struct {
		input  string
		arch string
		output *parsedOpcodes
	}
```
and if `nil` output means it must fail, then all test cases can be combined into a single table and runValidTests/runInvalidTests inlined into a single loop.